app/auth/README.md

```diff
@@ -0,0 +1,14 @@
+# Access Tokens
+All access tokens are valid for 30 minutes on the API. This can be adjusted by amending ACCESS_TOKEN_EXPIRE_MINUTES on the auth/security.py file. This can be authenticated via a username and password which is compared to the database. As long as the user is logged in, a JWT token can be generated for their user.
```

Comment:
Can you document which endpoints need to be called for the user to authenticate with the service?

And how the token should be used in subsequent requests please.